plugins/azure-devops-backend/src/api/AzureDevOpsApi.ts

```diff
@@ -397,12 +397,18 @@ export class AzureDevOpsApi {
       .filter((policy): policy is Policy => Boolean(policy));
   }
 
-  public async getAllTeams(): Promise<Team[]> {
+  public async getAllTeams(limit?: number): Promise<Team[]> {
```

Comment:
tiny suggestion to make the endpoint more evolvable in the future in case we add more options to the endpoint:

```suggestion
  public async getAllTeams(options?: { limit?: number }): Promise<Team[]> {
```